helm/prometheus-rules/templates/alerting-rules/apiserver.management-cluster.rules.yml

```diff
@@ -14,11 +14,11 @@ spec:
     # expr produces 95th percentile latency for requests to the api server, by request verb.
     # verb WATCH is excluded because it produces a flat line and is not relevant.
     # apiserver_request_latencies_bucket are expressed in microseconds, dividing by 1e+06 bring this to seconds.
-    - alert: ManagementClusterAPIServerLatencyTooHigh
+    - alert: ManagementClusterAPIServerLatencyTooHighCloud
       annotations:
         description: '{{`Kubernetes API Server {{ $labels.verb }} request latency is too high.`}}'
         opsrecipe: apiserver-overloaded/
-      expr: histogram_quantile(0.95, sum(rate(apiserver_request_latencies_bucket{verb=~"CONNECT|DELETE|GET|PATCH|POST|PUT"}[1h])) by (verb, le)) / 1e+06 > 1
+      expr: histogram_quantile(0.95, sum(rate(apiserver_request_duration_seconds_bucket{verb=~"CONNECT|DELETE|GET|PATCH|POST|PUT", provider=~"aws|azure|google"}[1h])) by (verb, le)) > 1
```

Comment:
Would be nice to have this templated so it is easier to change later on. But I would like to leave this for separate PR. @giantswarm/team-atlas any thoughts into this direction already?

Reply:
Sry I am speaking about this `, provider=~"aws|azure|google"`

Reply:
Oh. Can I make this simpler with this?

https://github.com/giantswarm/prometheus-rules/blob/master/helm/prometheus-rules/templates/_helpers.tpl#L31

Reply:
Alerts are routed based on the label?